src/erc721.js

```diff
@@ -150,6 +150,22 @@ export default class ERC721 {
       .catch((err) => console.log(err))
   }
 
+  getCollection(options = {}){
+    //tokenAddress=0xbb29a07c3f932188cc074500e41bcc39f9ac6bf4&apiKey=81546f88-431d-4f33-9045-f28c8df7c735&chain=ethereum&network=rinkeby
+    return axios
+      .request({
+        url: `${this.host}/api/v1/tokens/0xb08efdD94a2b705a12272Ae985Bc293d75aB70c9/collection?${qs.stringify(options)}`,
```

Comment:
This `0xb08efdD94a2b705a12272Ae985Bc293d75aB70c9` is a hardcoded Ethereum mainnet address?

Even if it is hardcoded, we may provide a variable name to it for reference.